iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -282,9 +282,12 @@ void PortManager::handleInterfaces() noexcept
         // also forward services from service registry
         /// @todo #415 do we still need this? yes but return a copy here to be stored in shared memory via new
         /// StatusPort's
+        /// @todo iox-#27 I guess this was necessary since a service could be offered via ServiceDiscovery;
+        /// this was removed and I somehow have the feeling this breaks the interface ports with the changes from this
+        /// PR if the CaproMessageSubType is something different than NOSUBTYPE
         auto serviceVector = m_serviceRegistry.getServices();
 
-        caproMessage.m_subType = capro::CaproMessageSubType::SERVICE;
+        caproMessage.m_subType = capro::CaproMessageSubType::NOSUBTYPE;
 
         for (auto const& element : serviceVector)
         {
```

Comment:
@mossmaurice are you okay with removing the code from below? #1084 will add the code for the `ServerPort`, which would be used to emulate methods for ara::com. Furthermore, the `ServiceDiscovery` cannot be used anymore to offer a service so this is currently dead code anyway.

Reply:
@elBoberido Well, this depends on the `InterfacePort`'s. As discussed in the in the last developer meetup, we won't remove them with `v2.0`. AFAIU the code would not be dead code till we have an alternative for the `InterfacePort`'s. Hence, please don't remove it.

Reply:
@mossmaurice it already is dead code. It was used to forward services to the `InterfacePort`'s. These were the ones from the `PublisherPort` or the ones which where offered by `PoshRuntime::offerService`, which is gone. The only other port which is offering is the new `ServerPort`.

Do I miss something?